test/intl402/Temporal/PlainDate/from/canonicalize-calendar.js

```diff
@@ -4,7 +4,7 @@
 /*---
 esid: sec-temporal.plaindate.from
 description: Calendar ID is canonicalized
-features: [Temporal]
+features: [Temporal, Intl.Era-monthcode]
```

Comment:
This one is technically not an Intl Era Monthcode feature, it's specified in [CanonicalizeCalendar](https://tc39.es/proposal-temporal/#sec-temporal-canonicalizecalendar)